src/iosanita/policy/upgrades/v1001.py

```diff
@@ -0,0 +1,72 @@
+from . import logger
+from collective.volto.blocksfield.field import BlocksField
+from plone import api
+from plone.dexterity.utils import iterSchemata
+from uuid import uuid4
+from zope.schema import getFields
+
+
+def upgrade(setup_tool=None):
+    """ """
+    brains = api.content.find(block_types="accordion")
+    tot = len(brains)
+    i = 0
+    for brain in brains:
+        i += 1
+        obj = brain.getObject()
+        logger.info("Upgrading accordion %s of %s: %s" % (i, tot, obj.absolute_url()))
+
+        blocks = getattr(obj, "blocks", {})
+        if blocks:
+            for uid, block in blocks.items():
+                new_accordion_block = fix_accordion(block=block)
+                if new_accordion_block:
+                    blocks[uid] = new_accordion_block
```

Comment:
You are changing data of a dict, are you sure that this change will be persistent in the zodb?